lldb/include/lldb/API/SBProgress.h

```diff
@@ -59,6 +63,9 @@ class LLDB_API SBProgress {
   lldb_private::Progress &ref() const;
 
 private:
+  SBProgress(const SBProgress &rhs) = delete;
```

Comment:
Oh! I didn't originally do this because I thought we weren't adding deletes in the header.

Reply:
SBStream does this to enforce no copying. :)